a3p-integration/proposals/a:upgrade-14/invite-submission/README.md

```diff
@@ -0,0 +1,12 @@
+This core-eval installs upgrades for the wallet and Zoe. This test verifies that
```

Comment:
Probably should remove the first sentence since the wallet factory and zoe upgrades are part of the chain software upgrade now.

Reply:
Please revise the whole readme. I think it was moved from a README for the earlier governance proposal but now it's scoped to just this core-eval used for `testRepairs`